src/components/Containers/TopSection/TopSection.jsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import styled from "styled-components";
+
+import SideGraphic from "../../../assets/top-section-graphic.svg";
+import { media } from "../../../utils";
+import { Text } from "../../Texts";
+import { Button } from "../../Inputs";
+
+export const TopSection = ({ ...props }) => (
+  <MainContainer {...props}>
+    <MobileGraphic>
+      <Graphic src={SideGraphic} alt="Graphic" />
+    </MobileGraphic>
```

Comment:
You have a duplicate of ```Graphic``` and so there's repetitive code. I would suggest using flex-box for the main container and look at flex directions. When it's mobile you can change the ```flex-direction``` property to reverse-column. Since there's no other function that's different about mobile and desktop, and just styling, we want to focus on changing it with css.

Reply:
Also alt should be slightly more descriptive in case the graphic cannot be retrieved